docs/pages/component/methods.mdx

```diff
@@ -141,8 +141,10 @@ const someCoolFunctions = async () => {
   videoRef.current.presentFullscreenPlayer();
   videoRef.current.dismissFullscreenPlayer();
 
-  // pause or play the video
-  videoRef.current.play();
+  // resume the video
+  videoRef.current.resume();
+
+  // pause the video
```

Comment:
The other methods examples are connected - eg. 
```tsx
  // present or dismiss fullscreen player
  videoRef.current.presentFullscreenPlayer();
  videoRef.current.dismissFullscreenPlayer();
  ```
  So I think that we also should connect those methods into one comment, so 
  ```tsx
  // pause or resume the video
  videoRef.current.pause();
  videoRef.current.resume();
```